Positions.xml

```diff
@@ -686,7 +686,7 @@
             </ArrivalLists>
             <ControllerInfo>
                 <Line>Torre San Jose / San Jose Tower</Line>
-                <Line>Serving MMSD and MMSL arrivals and departures</Line>
+                <Line>Serving MMSD arrivals and departures</Line>
```

Comment:
Shouldn't it be MMSL?

Reply:
Nevermind, different position.

Reply:
No, MMSD tower only covers MMSD
MMSL tower only covers MMSL

Changes are made on MMSD_TWR